<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabSudAuswahl</class>
 <widget class="QWidget" name="TabSudAuswahl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>491</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QRadioButton" name="rbAlle">
           <property name="text">
            <string>alle</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbNichtGebraut">
           <property name="text">
            <string>nicht gebraut</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbNichtAbgefuellt">
           <property name="text">
            <string>nicht abgefüllt</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbAbgefuellt">
           <property name="text">
            <string>abgefüllt</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbNichtVerbraucht">
           <property name="text">
            <string>nicht verbraucht</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbVerbraucht">
           <property name="text">
            <string>verbraucht</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbMerkliste">
           <property name="text">
            <string>Merkliste</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Braudatum zwischen</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DateEdit" name="tbDatumVon">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>und</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DateEdit" name="tbDatumBis">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbDatumAlle">
           <property name="text">
            <string>alle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="tbFilter">
           <property name="placeholderText">
            <string>Suche</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableSudauswahl">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Anzahl Sude:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblNumSude"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="verticalSpacing">
          <number>4</number>
         </property>
         <item row="0" column="2">
          <widget class="QPushButton" name="btnAlleVergessen">
           <property name="text">
            <string>Alle vergessen</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="btnLaden">
           <property name="text">
            <string>Sud laden</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="btnImportieren">
           <property name="text">
            <string>Sud importieren</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="btnAnlegen">
           <property name="text">
            <string>Neuen Sud anlegen</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="btnExportieren">
           <property name="text">
            <string>Sud exportieren</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btnVergessen">
           <property name="text">
            <string>Vergessen</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="btnMerken">
           <property name="text">
            <string>Merken</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="btnLoeschen">
           <property name="text">
            <string>Sud löschen</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="btnKopieren">
           <property name="text">
            <string>Sud kopieren</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="WebView" name="webview" native="true">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="cbEditMode">
           <property name="text">
            <string>Editiermodus</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ComboBox" name="cbTemplateAuswahl">
           <item>
            <property name="text">
             <string notr="true">sudinfo.html</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">style_hell.css</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">style_dunkel.css</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnSaveTemplate">
           <property name="text">
            <string>Speichern</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRestoreTemplate">
           <property name="text">
            <string>Wiederherstellen</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="tbTemplate">
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnToPdf">
         <property name="text">
          <string>PDF speichern</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WebView</class>
   <extends>QWidget</extends>
   <header>widgets/webview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DateEdit</class>
   <extends>QDateEdit</extends>
   <header>widgets/dateedit.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/combobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>rbAlle</tabstop>
  <tabstop>rbNichtGebraut</tabstop>
  <tabstop>rbNichtAbgefuellt</tabstop>
  <tabstop>rbAbgefuellt</tabstop>
  <tabstop>rbNichtVerbraucht</tabstop>
  <tabstop>rbVerbraucht</tabstop>
  <tabstop>cbMerkliste</tabstop>
  <tabstop>tbDatumVon</tabstop>
  <tabstop>tbDatumBis</tabstop>
  <tabstop>cbDatumAlle</tabstop>
  <tabstop>tbFilter</tabstop>
  <tabstop>tableSudauswahl</tabstop>
  <tabstop>btnMerken</tabstop>
  <tabstop>btnVergessen</tabstop>
  <tabstop>btnAlleVergessen</tabstop>
  <tabstop>btnAnlegen</tabstop>
  <tabstop>btnKopieren</tabstop>
  <tabstop>btnLoeschen</tabstop>
  <tabstop>btnImportieren</tabstop>
  <tabstop>btnExportieren</tabstop>
  <tabstop>btnLaden</tabstop>
  <tabstop>cbEditMode</tabstop>
  <tabstop>cbTemplateAuswahl</tabstop>
  <tabstop>btnSaveTemplate</tabstop>
  <tabstop>btnRestoreTemplate</tabstop>
  <tabstop>tbTemplate</tabstop>
  <tabstop>btnToPdf</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
